style.css

```diff
@@ -242,6 +243,10 @@ main > .main {
   align-items: center;
 }
 
+/* .work-section__card.active {
+  display: none;
+} */
+
```

Comment:
## [OPTIONAL]
- [ ] Kindly remove all your commented code to make your code cleaner.